pallets/registrar/src/lib.rs

```diff
@@ -749,11 +760,10 @@ pub mod pallet {
             genesis_data: ContainerChainGenesisData<T::MaxLengthTokenSymbol>,
         ) -> DispatchResult {
             let deposit = T::DepositAmount::get();
-
-            // Verify we can reserve
-            T::Currency::can_reserve(&account, deposit)
-                .then_some(true)
-                .ok_or(Error::<T>::NotSufficientDeposit)?;
+            // Verify we can hold
```

Comment:
The previous code also checked then reserve/hold, but why not hold directly?

Reply:
You're right, I guess we could hold directly and let it error if it can't succeed